reconciletags/config_integrity_test.py

```diff
@@ -0,0 +1,51 @@
+"""Tests to check the integrity of json config files.
+
+These tests assume that the json configs live in a top-level
+folder named config."""
+
+import glob
+import json
+import subprocess
+import unittest
+
+
+class ReconcilePresubmitTest(unittest.TestCase):
+
+    def _get_digests(self, repo):
+        output = json.loads(
+            subprocess.check_output(['gcloud', 'beta', 'container',
+                                     'images', 'list-tags',
+                                     '--no-show-occurrences',
+                                     '--format=json', repo]))
+        # grab the digest for each image and strip off the 'sha256:'
+        # for matching purposes
+        digests = [image['digest'][7:] for image in output]
```

Comment:
not a huge deal, but is it possible we ever get a digest generated with a different hash function? maybe it makes more sense (and is probably a bit cleaner) to just split the raw digest on a `:` and use everything that comes after that.